simple.js

```diff
@@ -1313,8 +1313,9 @@ define(function(require, exports, module) {
             // add Gravatar toggle online only
             info.getUser(addGravatarToggle);
 
-            // add C template
+            // add templates
             newresource.addFileTemplate(require("text!./templates/c.templates"), plugin);
+            newresource.addFileTemplate(require("text!./templates/py.templates"), plugin);
```

Comment:
It looks like we need to remove their menu item first? Otherwise, our template will try to [add](https://github.com/c9/c9.ide.newresource/blob/master/newresource.js#L155) a menu item with the same name, which would just get [ignored](https://github.com/c9/core/blob/master/plugins/c9.ide.ui/menus.js#L391)?